frontend/src/pages/modelRegistry/screens/ModelVersions/ModelVersionListView.tsx

```diff
@@ -54,7 +54,7 @@ const ModelVersionListView: React.FC<ModelVersionListViewProps> = ({
       <EmptyModelRegistryState
         testid="empty-model-versions"
         title="No versions"
-        description={`${rm?.name} has no versions registered to it. Register a version to this model.`}
+        description={`${rm?.name} has no registered versions. Register a version to this model.`}
```

Comment:
There was a discussion here about mentioning that there may be archived versions in this text, I'm not sure where that landed: https://docs.google.com/document/d/119Q_pgVphI-9HqCG3Uk6a1WJN5H8vYIr_sdosKDsXBA/edit?disco=AAABJv4aDms

In the newer doc here, there's instead an entirely separate new alert that appears when all versions have been archived: https://docs.google.com/document/d/1-YvEYVaVJXWKGGTsZTsVyxzKpaHYfSvGNzz6rIrlvDE/edit

I assume we should keep the text here "[model name] has no registered versions. Register a version to this model." and open a separate issue as part of https://issues.redhat.com/browse/RHOAIENG-10405 to add the new alert.

Reply:
> There was a discussion here about mentioning that there may be archived versions in this text, I'm not sure where that landed

The discussion is about how to show `view archived versions` button on the empty state, which the summit designs have already covered [here](https://www.figma.com/proto/KTsYhANYQqB0NUrXW7wQH1/Model-Registry?page-id=568%3A44455&node-id=5015-236832&viewport=1529%2C-594%2C0.17&t=pNQSsDACE9KAT3mB-1&scaling=min-zoom&content-scaling=fixed&starting-point-node-id=5015%3A234477). The `view archived versions` should only appear when there are archived versions for this model.

Reply:
> In the newer doc here, there's instead an entirely separate new alert that appears when all versions have been archived

The alert is part of the archive flow designs for MVP scope, check details [here](https://www.figma.com/proto/KTsYhANYQqB0NUrXW7wQH1/Model-Registry?page-id=6782%3A296930&node-id=6782-300265&viewport=654%2C233%2C0.09&t=7NEs0dflZy1yMJL4-1&scaling=min-zoom&content-scaling=fixed&starting-point-node-id=6782%3A300254). I've looked into the testing cluster and it seems that the alert hasn't been implemented for some reason. So I guess we just skip this alert for MVP and add it to the post-mvp effort.

Reply:
> The discussion is about how to show view archived versions button on the empty state
Ah, thanks. Yeah, that's already implemented.

> The alert is part of the archive flow designs for MVP scope, check details [here](https://www.figma.com/proto/KTsYhANYQqB0NUrXW7wQH1/Model-Registry?page-id=6782%3A296930&node-id=6782-300265&viewport=654%2C233%2C0.09&t=7NEs0dflZy1yMJL4-1&scaling=min-zoom&content-scaling=fixed&starting-point-node-id=6782%3A300254)
Oof, yeah, we overlooked that when we initially implemented that flow.

I want to stay focused on landing work for the sprint, but when I get a chance I'll open post-MVP issues for these things we missed from the MVP designs. I'd love help validating that we have everything logged after that, I'll get back to you.